lib/pg_party/adapter_decorator.rb

```diff
@@ -53,13 +53,13 @@ def create_table_like(table_name, new_table_name, **options)
       validate_primary_key(primary_key) unless create_with_pks
       if partition_type
         validate_supported_partition_type!(partition_type)
-        raise ArgumentError, '`partition_key` is required when specifying a partition_type' unless partition_key
+        raise ArgumentError, "`partition_key` is required when specifying a partition_type" unless partition_key
       end
 
       like_option = if !partition_type || create_with_pks
-                      'INCLUDING ALL'
+                      "INCLUDING ALL"
                     else
-                      'INCLUDING ALL EXCLUDING INDEXES'
+                      "INCLUDING ALL EXCLUDING INDEXES"
```

Comment:
I guess I missed this one previously, but I would prefer if everything was moved over to the left like I mentioned in another comment above